src/vm/compile.cpp

```diff
@@ -69,9 +69,11 @@
 #include "versionresilienthashcode.h"
 #include "inlinetracking.h"
 #include "jithost.h"
+#include "stubgen.h"
 
 #ifdef CROSSGEN_COMPILE
 CompilationDomain * theDomain;
+MapSHash<CORINFO_METHOD_HANDLE, CORINFO_METHOD_HANDLE> s_stubMethodsOfMethod;
```

Comment:
This should also be behind `FEATURE_READYTORUN_COMPILER`? I don't fully understand all the `#define`s here.

Reply:
Eh, probably. Will do.

Reply:
Fixed